Kernel/Storage/NVMe/NVMeController.cpp

```diff
@@ -47,11 +67,14 @@ ErrorOr<void> NVMeController::initialize()
     // Queues will individually map the doorbell register respectively
     m_controller_regs = TRY(Memory::map_typed_writable<volatile ControllerRegister>(PhysicalAddress(m_bar)));
 
+    auto caps = m_controller_regs->cap;
+    m_ready_timeout_ms = CAP_TO(caps) * 500; // CAP.TO is in 500ms units
```

Comment:
Would it be more expressive to use AK::Time and Time::from_milliseconds for this instead of storing integer Ms with _ms slapped on the end of the name? I might just make an enhancement issue for this, I'm sure there's other places that could use that pattern 🤔

Reply:
I thought about it, but the waiting loop seems much simpler this way

Reply:
Never mind, it's about the same. I changed to AK::Time instead